### ISSUE TITLE
Some animated layers do not render at all in the bodymovin export

### ISSUE THREAD

**Issue by AryToNeX** [airbnb/lottie-web#2465]
**Tell us about your environment**

* **Browser and Browser Version:** Microsoft Edge Chromium
* **After Effects Version:** CC 2020

**What did you do? Please explain the steps you took before you encountered the problem.**

I am trying to export an animation to JSON. However, a layer is missing from my animation.

**What did you expect to happen?**

I expected my animation to show up reliably.

**What actually happened? Please include as much _relevant_ detail as possible.**

An animated layer that is behind another animated layer does not show up at all.

**Please provide a download link to the After Effects file that demonstrates the problem.**
[Untitled Project 5.zip](https://github.com/airbnb/lottie-web/files/5976286/Untitled.Project.5.zip)

**Comment by bodymovin** [comment]
for some reason, one of your masks is set as ADDITIVE in AE even if it's not closed. So lottie tries to clip the shape and it ends up empty. If you recreate the mask it should work fine. You can try copying the mask from the other layer.

**Comment by AryToNeX** [comment]
redoing the mask from scratch fixed it, however, it might not be as easy with complex masks. there should be a way of toggling additive masks back so they render correctly imho

**Comment by bodymovin** [comment]
yes, but this looks more of an AE issue. I think that if you close the shape and reopen it, it might get fixed
